### PR TITLE
forum sniff TC

### PR DESCRIPTION
**Pull request by dmardassi**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java [hunk 1]
```diff
@@ -0,0 +1,746 @@
...
+    $(byText(nameCat)).should(Condition.exist);
+    $(byText(Topic)).should(Condition.exist);
+    info("Verify that the topic's activity is shown on intranet");
+    homePagePlatform.goToHomePage();
+    for (int i = 0; i <= 10; i++) {
+      refresh();
+      try {
+        Thread.sleep(1000);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java]: How do you deal with exception usually ? no logger ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java [hunk 2]
```diff
@@ -0,0 +1,746 @@
...
+    forumTopicManagement.selectItemMoreActionMenuTopic(ForumTopicManagement.specifMoreActionMenuTopic.MOVE);
+    info("Move the topic to a forum");
+    forumTopicManagement.moveTopicToForum(nameCat, nameForum);
+    info("Verify that the forum is moved to new category");
+    $(byText(Forum)).should(Condition.exist);
+    $(byText(nameCat)).should(Condition.exist);
+    $(byText(Topic)).should(Condition.exist);
+    info("Verify that the topic's activity is shown on intranet");
+    homePagePlatform.goToHomePage();
+    for (int i = 0; i <= 10; i++) {
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java]: Can't you move this piece of code in a method in order to reuse it (something like "refreshUntil(condition, timeout)") ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java (hunk 1 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java]: yes,I must log it ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java (hunk 2 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java]: ok (y)

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java (hunk 1 above)
Comment (thomasdelhomenie) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java]: How do you manage exceptions usually ?
Actually the best thing to do it maybe to just throw the exception (remove the try/catch and add throws Exception in the method declaration). WDYT ?

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java [hunk 3]
```diff
@@ -215,7 +216,19 @@ public void loadMoreActivities() {
     evt.waitForAndGetElement(ELEMENT_PLF_HOMEPAGE_LOAD_MORE_BUTTON, testBase.getDefaultTimeout(), 1);
     evt.click(ELEMENT_PLF_HOMEPAGE_LOAD_MORE_BUTTON);
   }
-
+public void refreshUntil(String text, Condition condition,long time){
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java]: Indentation is not good.
Also, add a blank line before and after the method.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java (hunk 1 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java]: Yes,I change it

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java (hunk 3 above)
Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java]: code formatted

**Pull request merged**